README.md

```diff
@@ -26,11 +28,11 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 BOOK STORE <a name="about-project"></a>
+# 📖 React Group Project <a name="about-project"></a>
 
-> THIS IS A REACT-REDUX PROJECT APP
+> THIS IS A REACT-REDUX GROUP PROJECT APP
```

Comment:
- [ ] [optional] It would be nice if you added more description about your project.